openslides/users/models.py

```diff
@@ -125,9 +127,19 @@ def get_short_name(self):
         E. g.: * Max Mustermann
                * Enders, Christoph
         """
-        # TODO: Order of name. See config.
-        name = ('%s %s' % (self.first_name, self.last_name)).strip()
-        return name or self.username
+        if self.first_name and self.last_name:
+            # The user has a last_name and a first_name
+            user_names = [self.last_name, self.first_name]
```

Comment:
What if the last name or the first name contains whitespaces at the beginning or at the end. They should be stripped away before combining the names.
Proposal also for more readability and less lines of code:
```python
if self.first_name and self.last_name:
    # The user has a last_name and a first_name
    if config['users_sort_users_by_first_name']:
        name = ' '.join(self.first_name.strip(), self.last_name.strip())
    else:
        name = ', '.join(self.last_name.strip(), self.first_name.strip())
elif ...
```